<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1078</width>
    <height>949</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background:#131239;</string>
  </property>
  <widget class="QPushButton" name="f_mole">
   <property name="geometry">
    <rect>
     <x>620</x>
     <y>220</y>
     <width>61</width>
     <height>61</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background: transparent;
border: none;</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="icon">
    <iconset>
     <normaloff>../Images and icon/Mole.png</normaloff>../Images and icon/Mole.png</iconset>
   </property>
   <property name="iconSize">
    <size>
     <width>60</width>
     <height>60</height>
    </size>
   </property>
  </widget>
  <widget class="QLabel" name="f_label">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>20</y>
     <width>71</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font-size:20px;
color: white;</string>
   </property>
   <property name="text">
    <string>SCORE:</string>
   </property>
  </widget>
  <widget class="QLabel" name="f_label_2">
   <property name="geometry">
    <rect>
     <x>240</x>
     <y>20</y>
     <width>71</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font-size:20px;
color: white;</string>
   </property>
   <property name="text">
    <string>TIME:</string>
   </property>
  </widget>
  <widget class="QLabel" name="f_score">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>20</y>
     <width>60</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font-size:20px;
color: white;</string>
   </property>
   <property name="text">
    <string>0</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="f_timer">
   <property name="geometry">
    <rect>
     <x>330</x>
     <y>20</y>
     <width>60</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font-size:20px;
color: white;</string>
   </property>
   <property name="text">
    <string>15</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="f_biglabel1">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>220</y>
     <width>401</width>
     <height>131</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font-size:40px;
color: white;</string>
   </property>
   <property name="text">
    <string>TIME UP</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="f_biglabel2">
   <property name="geometry">
    <rect>
     <x>360</x>
     <y>360</y>
     <width>91</width>
     <height>141</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font-size:80px;
color: white;</string>
   </property>
   <property name="text">
    <string>3</string>
   </property>
  </widget>
  <widget class="QPushButton" name="f_continue">
   <property name="geometry">
    <rect>
     <x>640</x>
     <y>710</y>
     <width>131</width>
     <height>51</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 15pt &quot;Arial Rounded MT Bold&quot;;
padding:15px;
background-color:green;
color:white;
border-radius: 20px;</string>
   </property>
   <property name="text">
    <string>NEXT</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
